packages/react-liveness/src/components/FaceLivenessDetector/shared/DefaultStartScreenComponents.tsx

```diff
@@ -1,39 +1,23 @@
 import React from 'react';
 import { ComponentClassName } from '@aws-amplify/ui';
-import { Flex, View, Text } from '@aws-amplify/ui-react';
-import { GoodFitIllustration, TooFarIllustration, StartScreenFigure } from './';
+import { Flex, View } from '@aws-amplify/ui-react';
+import { RecordingIcon } from './';
 import { LivenessIconWithPopover } from './LivenessIconWithPopover';
+import { CancelButton as CancelButtonComponent } from './CancelButton';
 import { LivenessClassNames } from '../types/classNames';
+import { CheckScreenComponents } from './FaceLivenessErrorModal';
+
+export type FaceLivenessDetectorComponents = StartScreenComponents &
+  CheckScreenComponents;
 
 export interface StartScreenComponents {
-  Header?: React.ComponentType;
   PhotosensitiveWarning?: React.ComponentType;
-  Instructions?: React.ComponentType;
-}
-
-interface DefaultHeaderProps {
-  headingText: string;
-  bodyText: string;
+  RecordingIcon?: React.ComponentType<DefaultRecordingIconProps>;
+  CancelButton?: React.ComponentType<CancelButtonProps>;
 }
 
-export const DefaultHeader = ({
-  headingText,
-  bodyText,
-}: DefaultHeaderProps): JSX.Element => {
-  return (
-    <View className={LivenessClassNames.StartScreenHeader}>
-      <View className={LivenessClassNames.StartScreenHeaderHeading}>
-        {headingText}
-      </View>
-      <View className={LivenessClassNames.StartScreenHeaderBody}>
-        {bodyText}
-      </View>
-    </View>
-  );
-};
-
 interface DefaultPhotosensitiveWarningProps {
-  headingText: string;
+  headingText?: string;
   bodyText: string;
   infoText: string;
```

Comment:
Why is headingText now optional?

Reply:
honestly not sure why i chagned this, good call, changing back